specs/latest/2.0/index.html

```diff
@@ -3344,6 +3515,23 @@ <h2 class="no-toc">Table of contents</h2>
         this limitation in WebGL 2 for all platforms.
     </div>
 
+    <h3><a name="DOM_UPLOAD_UNPACK_PARAMS">Pixel store parameters for uploads
+        from <code>TexImageSource</code></a></h3>
+    <p><code>UNPACK_ALIGNMENT</code> and <code>UNPACK_ROW_LENGTH</code> are
+       ignored, as they determine row-stride-in-bytes, which is immutable for
+       TexImageSource objects.
+    <p>Subrect selection is possible using <code>UNPACK_</code> params.
+       <code>UNPACK_SKIP_PIXELS</code> and <code>UNPACK_SKIP_ROWS</code>
+       determine the origin of the subrect, with <code>width</code> and
+       <code>height</code> determining the size of the subrect.
+    <p>For 3D textures, <code>UNPACK_SKIP_IMAGES</code> and
+       <code>UNPACK_IMAGE_HEIGHT</code> allow selection of multiple slices from
+       the 2D source.
+       Conceptually, <code>UNPACK_IMAGE_HEIGHT</code> determines how many rows
+       there are in an "image". For example, an element with a height of 20 and
+       <code>UNPACK_IMAGE_HEIGHT</code> set to 10 allows uploading data with
+       <code>depth</code> of 2.
```

Comment:
OK, I see now that this covers all of the texture-related entry points above -- if you feel strongly that this is the direction to take, then please reference this section from all of the entry points above taking TexImageSource. Maybe a sentence like: "Selection of a sub-rectangle of the source may be achieved as described in 'pixel storage parameters for uploads from TexImageSource' [link]".